recipe/meta.yaml

```
@@ -21,24 +21,24 @@ source:
   fn: wheel.whl
 
   url: https://pypi.io/packages/cp36/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-cp36-none-{{ linux_whl }}.whl  # [linux and py==36]
-  sha256: 0f83f307f3d5864a05dca7a70a667c4c7e7de3cdd5c116b1ec78fc18b6da57cc  # [linux and py==36]
+  sha256: 05858e34e542ed9fd54b066efa9a53ac6297eb5d6c1ee83f67808fc4d17a12bc  # [linux and py==36]
   url: https://pypi.io/packages/cp36/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-cp36-none-{{ osx_whl }}.whl  # [osx and py==36]
-  sha256: 13420851cb5245ed59f355d2b3f058f4725a421f7f4459e15eed549eafc16064  # [osx and py==36]
+  sha256: 4ad5c340b4f1a47011a8d499aa8590796532c2946691ca414c51e116223cf576  # [osx and py==36]
```

from https://pypi.org/project/jaxlib/#copy-hash-modal-434b4b85-ef8c-4a25-bc9b-c06114df1124

```suggestion
  sha256: f40ef7dd6bef4273f68fe5e724f29d0504ccc6b47b31ea19fab305fc4322a272  # [osx and py==36]
```